src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -473,6 +473,16 @@ void keyword_node( SummaryConfig::keyword_list& list,
                    ErrorGuard& errors,
                    const DeckKeyword& keyword)
 {
+    if (node_names.empty()) {
```

Comment:
I would hope you could use the existing `handleMissingNode()` function for this?

Reply:
> I would hope you could use the existing `handleMissingNode()` function for this

They're different conditions.  The `handleMissingNode()` function reports the case that a node-related summary quantity has been requested for a node that does not exist.  This check, on the other hand, fires if a node-related summary quantity has been requested in a run without network nodes at all.  The former is a [`SUMMARY_UNKNOWN_NODE`](https://github.com/OPM/opm-common/blob/3036441c8ccab915af5b23d234b49aa9f6ada273/opm/parser/eclipse/Parser/ParseContext.hpp#L289), the latter is a [`SUMMARY_UNHANDLED_KEYWORD`](https://github.com/OPM/opm-common/blob/3036441c8ccab915af5b23d234b49aa9f6ada273/opm/parser/eclipse/Parser/ParseContext.hpp#L290).